Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -365,39 +372,148 @@ def start_chrome_headless(chrome_port, chrome_binary=CHROME_EXE, user_options=""
     return None, None
 
 
-def terminate_chrome(browser):
-    global CHROME_PROCESS
-    demisto.debug(f'terminate_chrome, {CHROME_PROCESS=}')
+def terminate_chrome(chrome_port=None, killall=False):
+    processes = subprocess.check_output(['ps', 'auxww'], stderr=subprocess.STDOUT, text=True).splitlines()
+    chrome_renderer_identifiers = ["--type=renderer"]
```

Comment:
Are you sure it's needed?
isn't `--remote-debugging-port={chrome_port}` (or `--remote-debugging-port=` in the killall scenario) enough?

Reply:
Why was this marked as "resolved"?
Are you sure there are chrome processes with `--remote-debugging-port={chrome_port}` and `--type=renderer` ?

Reply:
I tried to change the function params to:
chrome_port='', killall=False
then 
chrome_identifiers = ["chrom", "headless", f"--remote-debugging-port={chrome_port}"]
works for the two situations: if there is chrome_port and if there is not.